solidity/contracts/KeepToken.sol

```diff
@@ -1,21 +1,22 @@
 pragma solidity 0.5.17;
 
 import "openzeppelin-solidity/contracts/token/ERC20/ERC20Burnable.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20Detailed.sol";
 
 
 /// @dev Interface of recipient contract for approveAndCall pattern.
 interface tokenRecipient { function receiveApproval(address _from, uint256 _value, address _token, bytes calldata _extraData) external; }
 
 /// @title KEEP Token
 /// @dev Standard ERC20Burnable token
-contract KeepToken is ERC20Burnable {
+contract KeepToken is ERC20Burnable, ERC20Detailed {
     string public constant NAME = "KEEP Token";
     string public constant SYMBOL = "KEEP";
     uint8 public constant DECIMALS = 18; // The number of digits after the decimal place when displaying token values on-screen.
     uint256 public constant INITIAL_SUPPLY = 10**27; // 1 billion tokens, 18 decimal places.
 
     /// @dev Gives msg.sender all of existing tokens.
-    constructor() public {
+    constructor() public ERC20Detailed(NAME, SYMBOL, DECIMALS) {
```

Comment:
`ERC20` already has a `(name, symbol)` constructor- what does ERC20Detailed get us?

Reply:
(I think, so many openzeppelin versions!)

Reply:
It doesn't, no: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v2.3.0/contracts/token/ERC20/ERC20.sol .

Reply:
(The most recent OpenZeppelin versions collapse a lot of the variants for some of the ERC tokens into a single contract, rather than offer 8 different possibilities.)

Reply:
Yep, [that was it](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol)

Reply:
It gives us the decimals detail. 

Apparently we're using the all-caps members (eg. `DECIMALS`) but they didn't do anything in the end.

Reply:
> (I think, so many openzeppelin versions!)

🤦this.